Fomo/User.swift

```diff
@@ -10,6 +10,18 @@ import UIKit
 var _currentUser: User?
 
 class User: NSObject {
+    
+    enum Response: Int {
+        case Accepted = 0, Declined, Maybe, Invited
+    }
+    
+    var id: Int?
+    var name: String?
+    var email: String?
+    var avatarImageUrl: String?
+    var preferences: [AttractionType]?
+    var response: Response?
+    
```

Comment:
should some sort of vote array (array of objects for each trip) be associated with the User? so we know what recommendation items they've seen and voted on